containers/ddev-webserver/ddev-webserver-base-files/usr/local/bin/ddev

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Imitation/replacement for `ddev` inside the web container
+# Partial functionality
+
+set -eu -o pipefail
+
+printf "You executed 'ddev $*' inside the web container\nbut many DDEV commands are not available.\n"
+
+if [ $# -ge 1 ] && command -v "$1" >/dev/null 2>&1; then
+  printf "\nHowever, you may be able to use a command like:\n\n$*\n".
```

Comment:
Removed the dot:

```suggestion
  printf "\nHowever, you may be able to use a command like:\n\n$*\n"
```

Because it was added here:

```
However, you may be able to use a command like:

drush
.stas@d11-web:/var/www/html$
```